01.general/01.adguard-ad-filters/docs.en.md

```diff
@@ -24,15 +24,15 @@ Our filter policy is available [here](https://kb.adguard.com/general/adguard-fil
 <a name="contribute"></a>
 ## Contributing to AdGuard filters
 
-We are blessed to have a community that does not only love AdGuard, but also gives back. A lot of people volunteer in various ways to make other users' experience with AdGuard better, and you can join them! We, on our part, can only be happy to reward the most active members of the community. So, what can you do?
+We are blessed to have a community that does not only love AdGuard, but also gives back. A lot of people volunteer in various ways to make other users' experiences with AdGuard better, and you can join them! We, for our part, can only be happy to reward the most active members of the community. So, what can you do?
 
 ### Report Issues
 
-We rely on the community to let us know about issues with our filters. This way we can be most efficient with our time and keep filters constantly updated. To submit a report, please use this [web reporting tool](https://agrd.io/report).
+We rely on the community to let us know about issues with our filters. This way, we can be much more efficient with our time and keep filters constantly updated. To submit a report, please use this [web reporting tool](https://agrd.io/report).
 
 ### Suggest Filtering Rules
 
-You will find a lot of open issues in the [GitHub filter repository](https://github.com/AdguardTeam/AdguardFilters/issues). Each references a problem with some website — a missed ad, a false positive etc. — choose any one and suggest your own rules in comments. AdGuard filter engineers will review your suggestions, and if they find them correct, your rules will be added to AdGuard filters.
+You will find a lot of open issues in the [GitHub filter repository](https://github.com/AdguardTeam/AdguardFilters/issues). Each references a problem with some website—a missed ad, a false positive, etc.—so choose any one and suggest your own rules in the comments. AdGuard filter engineers will review your suggestions, and if they find them correct, your rules will be added to AdGuard filters.
 
 Here is the [official documentation](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) on AdGuard filtering rules syntax. You'll need to read it before you'll be able to create your own filtering rules.
```

Comment:
```suggestion
Here is the [official documentation](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) on the syntax of AdGuard filtering rules. Please read it carefully; it will help you create your own filtering rules.
```